pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1452,4 +1452,24 @@ self: super: {
     };
   };
 
+  hcoord = overrideCabal super.hcoord (drv: {
+    # We need to use the git tag, as hackage tarball doesn't contain files necessary for checks:
+    # https://github.com/danfran/hcoord/issues/9
+    src = pkgs.fetchFromGitHub {
+      owner = "danfran";
+      repo = drv.pname;
+      rev = "v1.0.0.0";
+      sha256 = "16a7q2nz0k0z96yyyc9s5pblxaznb9mqj94k8md2a51f36f1vaw4";
+    };
```

Comment:
It would probably be easier to just mark this package as `dontCheck`, rather than overriding the upstream source.

The problem with overriding the upstream source is that we will have to remember to undo it at some point.  If we don't undo it, we will continue getting the old version.